CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ SC-8077 - the migration copy-parents-data-into-children-entities-and-delete-pare
 
 ### Added
 
+- SC-8030 - Setup orchestrator for deleting concept
```

Comment:
Sorry found one thing. 
Changelog is on wrong position, the Unreleased only changed to 25.3.0 without moving. This will sort your changelog entry to wrong position.